src/language/parser.js

```diff
@@ -635,3 +634,19 @@ function parseNamedType(parser): NamedType {
     loc: loc(parser, start)
   };
 }
+
+export var ParseOps = {
```

Comment:
yeah i could do that. i just think it's really annoying to have to export that many things. I would rather have a parser object with methods. no semantic changes. the parser object is mutable anyway and gets mutated anyways so I don't really see how its that different